docs/quick_tutorial/jinja2/setup.py

```diff
@@ -2,13 +2,22 @@
 
 requires = [
     'pyramid',
+    'pyramid_chameleon',
+    'pyramid_debugtoolbar',
+    'pyramid_jinja2',
     'waitress',
 ]
 
 setup(name='tutorial',
       install_requires=requires,
+      extras_require={
+          'test': [
+              'pytest',
+              'webtest',
+          ],
+      },
       entry_points="""\
       [paste.app_factory]
       main = tutorial:main
       """,
-)
\ No newline at end of file
+      )
```

Comment:
Why are all these closing parenthesis indented? They should not be.

Reply:
PyCharm has an inspection for PEP8. Either indentation form is acceptable.
https://www.python.org/dev/peps/pep-0008/#indentation